kore/src/Kore/Log/WarnStuckClaimState.hs

```diff
@@ -44,6 +44,13 @@ instance Pretty WarnStuckClaimState where
 
 instance Entry WarnStuckClaimState where
     entrySeverity _ = Warning
+    oneLineDoc (TermsUnifiableStuck claim) =
+        Pretty.vsep
+            ["TermsUnifiableStuck", Pretty.pretty @SourceLocation $ from claim]
+    oneLineDoc (TermsNotUnifiableStuck claim) =
+        Pretty.vsep
+            ["TermsNotUnifiableStuck", Pretty.pretty @SourceLocation $ from claim]
+
```

Comment:
Is there a reason to use `vsep` instead of `hsep` here? As a user, I would expect that each entry corresponds to exactly one line in the log.